app-admin/terraform/terraform-0.12.14.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit golang-base go-module
+
+DESCRIPTION="A tool for building, changing, and combining infrastructure safely"
+HOMEPAGE="https://www.terraform.io/"
+
+EGO_PN="github.com/hashicorp/${PN}"
+SRC_URI="https://github.com/hashicorp/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0 BSD-2 BSD-4 ECL-2.0 imagemagick ISC JSON MIT MIT-with-advertising MPL-2.0 unicode"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RESTRICT="test"
+
+DOCS=( {README,CHANGELOG}.md )
+
+src_prepare() {
+	default
+	sed -i -e "s/!=\s\+\"terraform\"/!=\ \"${P}\"/" \
```

Comment:
Sorry to return this to you once more, but this needs to `|| die`. Was this always here, I just spotted it?

Reply:
It hasn't always been there, as part of debugging the tests, a modification needed to be done to one of the generator scripts. I'll add a die but I'm inclined to leave it in, but just comment it out, with a note to myself